examples/examples.py

```diff
@@ -205,3 +205,11 @@ def to_int(arg):
 ])
 def extract_with_custom_transform_example(a_dictionary, my_param=None):
     return {}
+
+
+@extract(parameters=[
+    Parameter(path="/params/currency_example", func_param_name="a_dictionary",
+              validators=[CurrencyValidator()])
```

Comment:
Since validate is static, we can just pass the class `CurrencyValidator` in as a validator without instantiating it.

```py
@extract(parameters=[
    Parameter(path="/params/currency_example", func_param_name="a_dictionary",
              validators=[CurrencyValidator])
```

This applies to all of the currency validator unit tests that don't have a custom error message

Reply:
I have changed those but test_currency_validator_returns_false_when_invalid_code_passed_in fails after passing in the class, not quite sure why that is. I'll look into it tomorrow.

Reply:
I've just had a look and it fails because the error message contains a reference to the {value}, which cannot be substituted, as the code currently stands, when called statically.

You can change line 123 from:

errors.append(validator.ERROR_MESSAGE)

to: 

errors.append(validator.ERROR_MESSAGE.format(value=value))

and that should work.

Reply:
it is just that until now, we had no validator that could be called statically and cared about the value when displaying the error.

Reply:
after fixing this, there'll also be some missing coverage as you'll probably have to test that the validator can be called in a non-static way.

Reply:
The change to class.py resolved the issue and I have added a new test to check the validate can be called non-statically. I have also added .coveragerc and updated the CONTRIBUTING.md to reflect the changes.

Reply:
@jamsidedown  I have made the requested changes. Awaiting your review.